<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACAudioControlsDockWidgetQt</class>
 <widget class="QDockWidget" name="ACAudioControlsDockWidgetQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>260</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetClosable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Audio Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>200</height>
    </size>
   </property>
   <widget class="QWidget" name="layoutWidgetGlobal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutGlobal">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBoxFeedback">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>231</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>175</height>
        </size>
       </property>
       <property name="title">
        <string>Audio Feedback</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="layoutWidgetFeedback">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>251</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutFeedback">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonMuteAll">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Mute All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxModes">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>231</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>231</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Playback modes</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <widget class="QLabel" name="labelForce">
            <property name="geometry">
             <rect>
              <x>184</x>
              <y>22</y>
              <width>25</width>
              <height>25</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Force</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelSynchroMode">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>47</y>
              <width>40</width>
              <height>25</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Synchro</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBoxScaleMode">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>56</x>
              <y>70</y>
              <width>119</width>
              <height>25</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
           <widget class="QLabel" name="labelScaleMode">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>72</y>
              <width>40</width>
              <height>25</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBoxPlaybackPreset">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>56</x>
              <y>20</y>
              <width>119</width>
              <height>25</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Sync Looping</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>High-Fidelity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Custom</string>
             </property>
            </item>
           </widget>
           <widget class="QCheckBox" name="checkBoxSynchroMode">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>182</x>
              <y>45</y>
              <width>18</width>
              <height>20</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;unchecked&lt;/span&gt;: default mode per loop, but if tempo is detected in a loop, the &amp;quot;&lt;span style=&quot; font-weight:600;&quot;&gt;AutoBeat&lt;/span&gt;&amp;quot; mode is used for the given loop&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;checked&lt;/span&gt;: forced for all loops&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBoxScaleMode">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>182</x>
              <y>70</y>
              <width>18</width>
              <height>20</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;unchecked&lt;/span&gt;: default mode per loop, but if tempo is detected in a loop, the &amp;quot;&lt;span style=&quot; font-weight:600;&quot;&gt;Vocode&lt;/span&gt;&amp;quot; mode is used for the given loop&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;checked&lt;/span&gt;: forced for all loops&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="labelPreset">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>22</y>
              <width>40</width>
              <height>25</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Preset</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBoxSynchroMode">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>56</x>
              <y>45</y>
              <width>119</width>
              <height>25</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxVisualization">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>75</height>
        </size>
       </property>
       <property name="title">
        <string>Audio Visualization</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QGroupBox" name="groupBoxWaveform">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>231</width>
          <height>55</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>211</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="title">
         <string>Waveform</string>
        </property>
        <widget class="QWidget" name="layoutWidgetWaveform">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>191</width>
           <height>26</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayoutWaveform">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelWaveformBrowser">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Browser</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxWaveformBrowser">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Classic</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
